package.json

```diff
@@ -66,7 +66,7 @@
     "@types/redux": "3.6.0",
     "@types/redux-localstorage": "1.0.8",
     "@types/rimraf": "2.0.3",
-    "@types/webdriverio": "4.13.3",
+    "@types/webdriverio": "5.0.0",
```

Comment:
according to the `package.json` in `spectron`, version 9 of `spectron` (necessary for `electron@7`) still uses `webdriverio@4` so we will want to undo this change.

the good news is that starting with `spectron@10` it pulls in the `webdriver` and `@types/webdriverio` packages for us 👍

Reply:
reverted version bump in 65c5288